web/src/components/organisms/Detail.jsx

```diff
@@ -0,0 +1,16 @@
+import { FaCalendarAlt, FaMapMarkerAlt } from 'react-icons/fa';
+
+const Detail = ({ label, value }) => {
+    return (
+        <div className="flex justify-between items-center py-3 border-b border-gray-200">
+            <span className="font-semibold">{label}:</span>
+            <span className="flex items-center">
+                {label === 'Date' && <FaCalendarAlt className="mr-2" />}
+                {label === 'Branch' && <FaMapMarkerAlt className="mr-2" />}
+                {value}
+            </span>
+        </div>
+    );
+};
```

Comment:
1. You can pass icon like this, no need to manually identify
2. add margin on the `value`

```suggestion
const Detail = ({ label, value, icon }) => {
    return (
        <div className="flex justify-between items-center py-3 border-b border-gray-200">
            <span className="font-semibold">{label}:</span>
            <span className="flex items-center">
                {!!icon && icon }
                {value}
            </span>
        </div>
    );
};
```

Then in detail it would look like this:
```
<Detail
...
icon={<FaCalendarAlt />}
/>
```